helm-chart-sources/datafabrikken-veiviser-dele-data/templates/deployment.yaml

```
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    datafabrikken.service: datafabrikken-veiviser-dele-data
+  name: datafabrikken-veiviser-dele-data
+  namespace: {{ required "Missing value NAMESPACE" .Values.NAMESPACE }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      datafabrikken.service: datafabrikken-veiviser-dele-data
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        datafabrikken.service: datafabrikken-veiviser-dele-data
+    spec:
+      containers:
+        name: datafabrikken-veiviser-dele-data
+        image: {{ required "Missing value DOCKER_IMAGE_NAME" .Values.DOCKER_IMAGE_NAME }}
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 8080
```

Tror kanskje det feiler uten tab her
```suggestion
          - containerPort: 8080
```

---

Det er gjort slik mange andre steder så tror det går fint.